Configs/.config/hypr/hyprland.conf

```diff
@@ -24,19 +24,20 @@ monitor = ,preferred,auto,auto
 
 # See https://wiki.hyprland.org/Configuring/Keywords/
 
-exec-once = $scrPath/resetxdgportal.sh # reset XDPH for screenshare
-exec-once = dbus-update-activation-environment --systemd WAYLAND_DISPLAY XDG_CURRENT_DESKTOP # for XDPH
-exec-once = dbus-update-activation-environment --systemd --all # for XDPH
-exec-once = systemctl --user import-environment WAYLAND_DISPLAY XDG_CURRENT_DESKTOP # for XDPH
-exec-once = $scrPath/polkitkdeauth.sh # authentication dialogue for GUI apps
-exec-once = waybar # launch the system bar
-exec-once = blueman-applet # systray app for Bluetooth
-exec-once = nm-applet --indicator # systray app for Network/Wifi
-exec-once = dunst # start notification demon
-exec-once = wl-paste --type text --watch cliphist store # clipboard store text data
-exec-once = wl-paste --type image --watch cliphist store # clipboard store image data
-exec-once = $scrPath/swwwallpaper.sh # start wallpaper daemon
-exec-once = $scrPath/batterynotify.sh # battery notification
+exec-once = $scrPath/resetxdgportal.sh # Reset XDPH for screenshare
+exec-once = dbus-update-activation-environment --systemd WAYLAND_DISPLAY XDG_CURRENT_DESKTOP # For XDPH
+exec-once = dbus-update-activation-environment --systemd --all # For XDPH
+exec-once = systemctl --user import-environment WAYLAND_DISPLAY XDG_CURRENT_DESKTOP # For XDPH
+exec-once = $scrPath/polkitkdeauth.sh # Authentication dialogue for GUI apps
+exec-once = waybar # Launch the system bar
+exec-once = blueman-applet # Systray app for Bluetooth
+exec-once = udiskie # Front-end that allows to manage removable media
+exec-once = nm-applet --indicator # Systray app for Network/Wifi
+exec-once = dunst # Start notification demon
+exec-once = wl-paste --type text --watch cliphist store # Clipboard store text data
+exec-once = wl-paste --type image --watch cliphist store # Clipboard store image data
+exec-once = $scrPath/swwwallpaper.sh # Start wallpaper daemon
+exec-once = $scrPath/batterynotify.sh # Battery notification
```

Comment:
Thanks for adding the comment, but can you make the comments here all start lowercase like it currently is? It's not necessary, it's just for consistency with the rest of the project. I'll approve it when that's done!

Reply:
sure thing